go.mod

```diff
@@ -0,0 +1,16 @@
+module github.com/mendersoftware/go-lib-micro
+
+go 1.14
+
+require (
+	github.com/ant0ine/go-json-rest v3.3.2+incompatible
```

Comment:
this is ce4b71cdf7ba29ef443d704bd923f5bbf281ee30 (Wed Nov 30 19:48:30 2016 -0800), but what we use in go-lib-micro is ebb33769ae013bd5f518a8bac348c310dea768b8 (Wed Sep 13 04:12:08 2017 +0000) why the change?

Reply:
This is just the auto-filled version (by go mod init), which defaults to the latest recognized semver tag. I suppose I can update this to the latest commit on master.